frontend/src/app/shared/models/user-prefferences/new-user-preferences.ts

```diff
@@ -0,0 +1,8 @@
+export interface INewUserPreferences {
+    languageId?: number | null;
```

Comment:
Are you sure "| null" is needed here?

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/68563967/43c29d23-cd4e-4742-bba7-c45439d6b7f1)

probably yes because the method should return string | null
and when i write field without | null , i get an exeption🤕

Reply:
removed